### PR TITLE
ENH: add --skip_happi flag

### LINKED ISSUE #116: Add Flag to Skip Happi Beamline Loading
In the timing system we don't really have a beamline, per-se, but still use a LUCID screen to aggregate all of our various launchers. Because we don't have any beamline devices defined in Happi, we get the following error message everytime we open the screen: 

image

It would be nice if there was a command flag we could use to skip this. We have to clear the message every time, and this sometimes causes confusion. Occasionally the error message will pop up behind other screens, making it appears as though the screen has frozen. The LUCID screen cannot be interacted with until this error window is closed. 

I think this could be done fairly easily in the HappiLoader run method: https://github.com/pcdshub/lucid/blob/master/lucid/launcher.py#L144

### PR DESCRIPTION
**Pull request by tangkong**
## Description
adds `--skip_happi` flag to the `lucid` cli, defaulting to false.  Allows `lucid` to be used for just the toolbar, rather than the grid of devices.

## Context
closes #116 

## How has this been tested?
interactively

### CODE REVIEW

File: lucid/launcher.py [hunk 1]
```diff
@@ -78,6 +78,12 @@ def get_parser():
         action='store_true',
         default=False
     )
+    parser.add_argument(
+        '--skip_happi',
```
Comment (klauer) [inline lucid/launcher.py:82]: Consistency-wise this argument makes sense (above arguments are in the style `--a_b`)
In the future, it'd be nice to switch to `--a-b` as it's more ergonomic from a CLI user standpoint

File: lucid/launcher.py [hunk 2]
```diff
@@ -215,7 +224,8 @@ def launch(beamline, *, toolbar=None, row_group_key="location_group",
     cbs = [grid.add_from_dict]
     loader = HappiLoader(beamline=beamline,
```
Comment (klauer) [inline lucid/launcher.py:225]: A flag to a loader saying "don't do anything" seems a bit of an anti-pattern (a separate `NoOpLoader` might have been cleaner?)

**Review by klauer — APPROVED** [review]
LUCID itself is a larger mess than the above nitpicks, so in the interest of getting this deployed 👍 from me

File: lucid/launcher.py (hunk 1 above)
Comment (tangkong) [reply lucid/launcher.py:82]: I had the same thought, but wasn't sure if it was just a flight of whimsy on my end. 👍

File: lucid/launcher.py (hunk 2 above)
Comment (tangkong) [reply lucid/launcher.py]: Good point 👍

**Review by ZLLentz — APPROVED** [review]
I'm interested to hear the story behind this one

**Comment by klauer** [comment]
> I'm interested to hear the story behind this one

➡️  https://github.com/pcdshub/lucid/issues/116

**Comment by tangkong** [comment]
I shall merge!  This was a quick fix inspired by a detailed issue and stronger-than-usual cup of coffee.

**Pull request merged**